content/en/docs/reference/instrumentation/zpages.md

```diff
@@ -0,0 +1,61 @@
+---
+reviewers:
+- richabanker
+title: Kubernetes z-pages
+content_type: reference
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.32" state="alpha" >}}
+
+Kubernetes core components expose a suite of z-endpoints to make it easier for users to debug their components. As of 1.32, these endpoints only support plain text format and are strictly to be used for human inspection to gain real time debugging information of a component binary. Avoid automated scraping of data returned by these endpoints since they are introduced as alpha features and their format may change in future releases.  
+
+<!-- body -->
+
+## z-pages
+
+Kubernetes now includes "z-pages" to help you troubleshoot problems with its core control plane components. These special debugging endpoints provide internal information about running components. Currently we have the following endpoints:
+
+- [z-pages](#z-pages)
+  - [statusz](#statusz)
+  - [flagz](#flagz)
+
+### statusz
+
+Enabled using the ComponentStatusz feature gate, statusz endpoint displays the high level information about the component such as its Kubernetes version, emulation version, start time and more.
+
+The statusz data for apiserver looks like this:
+
+```
+kube-apiserver statusz
+Warning: This endpoint is not meant to be machine parseable, has no formatting compatibility guarantees and is for debugging purposes only.
+
+Started: Wed Oct 16 21:03:43 UTC 2024
+Up: 0 hr 00 min 16 sec
+Go version: go1.23.2
+Binary version: 1.32.0-alpha.0.1484&#43;5eeac4f21a491b-dirty
+Emulation version: 1.32.0-alpha.0.1484
+```
+
+### flagz
+
+Enabled using the ComponentFlagz feature gate, the flagz endpoint shows you the command line flags that were used to start a component.
```

Comment:
```suggestion
Enabled using the `ComponentFlagz` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/), the `/flagz` endpoint shows you the command line arguments that were used to start a component.
```

Reply:
Although Go calls all command line arguments "flags", this isn't true across the world of IT. In documentation (not just K8s) I consistently call for the more widely recognized term. We can still talk about flagz when talking about the endpoint.